pkg/querier/http.go

```diff
@@ -141,6 +146,37 @@ func (q *QueryHandlers) Render(w http.ResponseWriter, req *http.Request) {
 		}
 	}
 
+	format := req.URL.Query().Get("format")
+	if format == "dot" {
+		// We probably should distinguish max nodes of the source pprof
+		// profile and max nodes value for the output profile in dot format.
+		sourceProfileMaxNodes := int64(512)
+		dotProfileMaxNodes := int64(100)
```

Comment:
This is quite important: the less nodes we include into the source, the higher level function will appear in the analysis